### PR TITLE
Suggested edits

### PR DESCRIPTION
**Pull request by heath-hamilton**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@heath-hamilton : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ide/quickstart-nodejs.md [hunk 1]
```diff
@@ -15,94 +15,115 @@ dev_langs:
...
 
-   - The npm node shows any installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
+   - The **npm** node shows installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
 
 1. If you want to install npm packages or Node.js commands from a command prompt, right-click the project node and choose **Open Command Prompt Here**.
 
    ![Node.js command prompt](../ide/media/quickstart-nodejs-command-prompt.png)
 
-1. In the *server.js* file in the editor (left pane), choose `http.createServer` and then press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *index.d.ts*.
+1. In the open *server.js* file, select `http.createServer` and press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *index.d.ts*.
```
Comment (heath-hamilton) [inline docs/ide/quickstart-nodejs.md:102]: I could be missing something, but when I went through the quickstart, the `createServer` function was in my *http.d.ts* file and not an *index.d.ts* file. Also, should we explain what this step is for? It isn’t clear to me.

File: docs/ide/quickstart-nodejs.md [hunk 2]
```diff
@@ -15,94 +15,115 @@ dev_langs:
...
-1. Press **Ctrl**+**F5** (or **Debug > Start Without Debugging**) to run the application. The app opens in a browser.
+## Run the app
 
-1. In the browser window, you will see "Hello World" plus the local port number.
+1. Press **Ctrl**+**F5** (or **Debug > Start Without Debugging**) to run the app. 
+ 
+   The app opens in a browser.
 
-1. Close the web browser.
+1. In the browser, verify that you see a "Hello World" message and the local port number.
```
Comment (heath-hamilton) [inline docs/ide/quickstart-nodejs.md:124]: Should we provide more detail about what port number the user should expect to see?

File: docs/ide/quickstart-nodejs.md (hunk 1 above)
Comment (Mikejo5000) [inline docs/ide/quickstart-nodejs.md:102]:
```suggestion
1. In the open *server.js* file, select `http.createServer` and press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *http.d.ts*.
```

**Comment by PRMerger7** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/ide/quickstart-nodejs.md (hunk 2 above)
Comment (Mikejo5000) [reply docs/ide/quickstart-nodejs.md:124]: I think it's OK as-is. It's not something that will change unless they change it explicitly.

File: docs/ide/quickstart-nodejs.md [hunk 3]
```diff
@@ -15,94 +15,115 @@ dev_langs:
...
 
-   - The npm node shows any installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
+   - The **npm** node shows installed npm packages. You can right-click the npm node to search for and install npm packages using a dialog box.
 
 1. If you want to install npm packages or Node.js commands from a command prompt, right-click the project node and choose **Open Command Prompt Here**.
 
    ![Node.js command prompt](../ide/media/quickstart-nodejs-command-prompt.png)
 
-1. In the *server.js* file in the editor (left pane), choose `http.createServer` and then press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *index.d.ts*.
+1. In the open *server.js* file, select `http.createServer` and press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *http.d.ts*.
```
Comment (Mikejo5000) [inline docs/ide/quickstart-nodejs.md:102]:
```suggestion
1. If you want to test navigation to source code, from the open *server.js* file, select `http.createServer` and press **F12** or choose **Go To Definition** from the context (right-click) menu. This command takes you to the definition of the `createServer` function in *http.d.ts*.
```

**Comment by PRMerger9** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Mikejo5000 — APPROVED** [review]
Thanks! Changes look good.

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**